monitoring/cluster_observability_operator/cluster-observability-operator-release-notes.adoc

```diff
@@ -12,12 +12,40 @@ include::snippets/technology-preview.adoc[leveloffset=+2]
 
 The {coo-first} is an optional {product-title} Operator that enables administrators to create standalone monitoring stacks that are independently configurable for use by different services and users.
 
-The {coo-short} complements the built-in monitoring capabilities of {product-title}. You can deploy it in parallel with the default platform and user workload monitoring stacks managed by the Cluster Monitoring Operator (CMO). 
+The {coo-short} complements the built-in monitoring capabilities of {product-title}. You can deploy it in parallel with the default platform and user workload monitoring stacks managed by the Cluster Monitoring Operator (CMO).
 
 These release notes track the development of the {coo-full} in {product-title}.
 
+[id="cluster-observability-operator-release-notes-0-1-2"]
+== {coo-full} 0.1.2
+The following advisory is available for {coo-full} 0.1.2:
+
+* link:https://access.redhat.com/errata/RHEA-2024:127118[2024:127118 Cluster Observability Operator 0.1.2]
+
+[id="cluster-observability-operator-0-1-2-CVEs"]
+=== CVEs
+
+* link:https://access.redhat.com/security/cve/CVE-2023-45142[CVE-2023-45142]
+
+[id="cluster-observability-operator-0-1-2-bug-fixes"]
+=== Bug fixes
+
+* Previously, certain cluster service version (CSV) annotations were not included in the metadata for {coo-short}.
+Because of these missing annotations certain {coo-short} features and capabilities did not appear in the package manifest or in the OperatorHub user interface.
```

Comment:
Nit: https://issues.redhat.com/browse/COO-12 added an annotation true but this enables automated updates where a newer version replaces an older one. Before this the older version would stay installed.

Might be worth calling this out?

Reply:
Added some verbiage to describe this change.